build/bump_version.py

```
@@ -0,0 +1,112 @@
+#!/usr/bin/env python3
+import subprocess
+import sys
+import re
+import os
+
+
+def get_files(paths=['.', 'premium']):
```

Ruff

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)